Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -1045,51 +1078,51 @@ end Anisotropic
+#align quadratic_form.pos_def QuadraticMap.PosDef
 
-theorem PosDef.smul {R} [LinearOrderedCommRing R] [Module R M] {Q : QuadraticForm R M}
+theorem PosDef.smul {R} [LinearOrderedCommRing R] [Module R M] {Q : QuadraticMap R M R}
     (h : PosDef Q) {a : R} (a_pos : 0 < a) : PosDef (a • Q) := fun x hx => mul_pos a_pos (h x hx)
-#align quadratic_form.pos_def.smul QuadraticForm.PosDef.smul
+#align quadratic_form.pos_def.smul QuadraticMap.PosDef.smul
 
 variable {n : Type*}
 
-theorem PosDef.nonneg {Q : QuadraticForm R₂ M} (hQ : PosDef Q) (x : M) : 0 ≤ Q x :=
+theorem PosDef.nonneg {Q : QuadraticMap R₂ M R₂} (hQ : PosDef Q) (x : M) : 0 ≤ Q x :=
   (eq_or_ne x 0).elim (fun h => h.symm ▸ (map_zero Q).symm.le) fun h => (hQ _ h).le
-#align quadratic_form.pos_def.nonneg QuadraticForm.PosDef.nonneg
+#align quadratic_form.pos_def.nonneg QuadraticMap.PosDef.nonneg
 
-theorem PosDef.anisotropic {Q : QuadraticForm R₂ M} (hQ : Q.PosDef) : Q.Anisotropic := fun x hQx =>
-  by_contradiction fun hx =>
+theorem PosDef.anisotropic {Q : QuadraticMap R₂ M R₂} (hQ : Q.PosDef) : Q.Anisotropic :=
+  fun x hQx => by_contradiction fun hx =>
     lt_irrefl (0 : R₂) <| by
       have := hQ _ hx
       rw [hQx] at this
       exact this
-#align quadratic_form.pos_def.anisotropic QuadraticForm.PosDef.anisotropic
+#align quadratic_form.pos_def.anisotropic QuadraticMap.PosDef.anisotropic
 
-theorem posDef_of_nonneg {Q : QuadraticForm R₂ M} (h : ∀ x, 0 ≤ Q x) (h0 : Q.Anisotropic) :
+theorem posDef_of_nonneg {Q : QuadraticMap R₂ M R₂} (h : ∀ x, 0 ≤ Q x) (h0 : Q.Anisotropic) :
     PosDef Q := fun x hx => lt_of_le_of_ne (h x) (Ne.symm fun hQx => hx <| h0 _ hQx)
-#align quadratic_form.pos_def_of_nonneg QuadraticForm.posDef_of_nonneg
+#align quadratic_form.pos_def_of_nonneg QuadraticMap.posDef_of_nonneg
 
-theorem posDef_iff_nonneg {Q : QuadraticForm R₂ M} : PosDef Q ↔ (∀ x, 0 ≤ Q x) ∧ Q.Anisotropic :=
+theorem posDef_iff_nonneg {Q : QuadraticMap R₂ M R₂} : PosDef Q ↔ (∀ x, 0 ≤ Q x) ∧ Q.Anisotropic :=
   ⟨fun h => ⟨h.nonneg, h.anisotropic⟩, fun ⟨n, a⟩ => posDef_of_nonneg n a⟩
-#align quadratic_form.pos_def_iff_nonneg QuadraticForm.posDef_iff_nonneg
+#align quadratic_form.pos_def_iff_nonneg QuadraticMap.posDef_iff_nonneg
 
-theorem PosDef.add (Q Q' : QuadraticForm R₂ M) (hQ : PosDef Q) (hQ' : PosDef Q') :
+theorem PosDef.add (Q Q' : QuadraticMap R₂ M R₂) (hQ : PosDef Q) (hQ' : PosDef Q') :
     PosDef (Q + Q') := fun x hx => add_pos (hQ x hx) (hQ' x hx)
-#align quadratic_form.pos_def.add QuadraticForm.PosDef.add
+#align quadratic_form.pos_def.add QuadraticMap.PosDef.add
 
 theorem linMulLinSelfPosDef {R} [LinearOrderedCommRing R] [Module R M] (f : M →ₗ[R] R)
     (hf : LinearMap.ker f = ⊥) : PosDef (linMulLin f f) := fun _x hx =>
   mul_self_pos.2 fun h => hx <| LinearMap.ker_eq_bot'.mp hf _ h
-#align quadratic_form.lin_mul_lin_self_pos_def QuadraticForm.linMulLinSelfPosDef
+#align quadratic_form.lin_mul_lin_self_pos_def QuadraticMap.linMulLinSelfPosDef
 
 end PosDef
```

Comment:
- This section should be in the namespace `QuadraticForm` instead of `QuadraticMap`.
- Every occurrence of `QuadraticMap` in this section should be replaced with `QuadraticForm`.

Reply:
I've adapted the `PosDef` section so that the results now work for Quadratic Maps.